### PR TITLE
Oscillator strength for naive/projected/allprojected qLR with Qiskit interface

### PR DESCRIPTION
**Pull request by ziechys**

### CODE REVIEW

**Comment by ziechys** [comment]
Renaming of `one_elec_op_pauli_0i_0a` not included yet

File: slowquant/qiskit_interface/linear_response/allprojected.py [hunk 1]
```diff
@@ -1,11 +1,21 @@
+from collections.abc import Sequence
+
 import numpy as np
 from dmdm.util import iterate_t1_sa, iterate_t2_sa  # temporary solution
```
Comment (erikkjellgren) [inline slowquant/qiskit_interface/linear_response/allprojected.py:4]:
```suggestion
```

File: slowquant/qiskit_interface/linear_response/allprojected.py [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 
+from slowquant.molecularintegrals.integralfunctions import (
+    one_electron_integral_transform,
+)
+from slowquant.qiskit_interface.base import FermionicOperator
```
Comment (erikkjellgren) [inline slowquant/qiskit_interface/linear_response/allprojected.py:9]:
```suggestion
```

File: slowquant/qiskit_interface/linear_response/projected.py [hunk 3]
```diff
@@ -1,10 +1,19 @@
+from collections.abc import Sequence
+
 import numpy as np
+
+from slowquant.molecularintegrals.integralfunctions import (
+    one_electron_integral_transform,
+)
+from slowquant.qiskit_interface.base import FermionicOperator
```
Comment (erikkjellgren) [inline slowquant/qiskit_interface/linear_response/projected.py:8]:
```suggestion
```

**Comment by ziechys** [comment]
Renaming addded

**Pull request merged**